projects/aca-content/assets/app.extensions.json

```diff
@@ -295,7 +296,7 @@
             "order": 200,
             "title": "APP.NEW_MENU.MENU_ITEMS.CREATE_LIBRARY",
             "description": "APP.NEW_MENU.TOOLTIPS.CREATE_LIBRARY",
-            "icon": "create_new_folder",
+            "icon": "adf:create_library",
```

Comment:
where this icon is coming from? I don't see this registered anywhere

Reply:
this has been changed to use pre-existing icons